apps/condo/domains/organization/tasks/syncSbbolTransactions.js

```diff
@@ -0,0 +1,37 @@
+const { createTask, createCronTask } = require('@open-condo/keystone/tasks')
+const { User } = require('@condo/domains/user/utils/serverSchema')
+const { requestTransactions } = require('@condo/domains/organization/integrations/sbbol/sync/requestTransactions')
+const { SBBOL_IMPORT_NAME } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { getLogger } = require('@open-condo/keystone/logging')
+const dayjs = require('dayjs')
+const isEmpty = require('lodash/isEmpty')
+
+const logger = getLogger('sbbol/CronTaskSyncTransactions')
+
+/**
+ * Synchronizes SBBOL transaction data with data in the system
+ * @returns {Promise<void>}
+ */
+async function syncSbbolTransactions (date, userId) {
+    // if userId is passed, receive transactions only for it. Case when it's not a cron task
+    if (userId) return await requestTransactions(userId, date)
+
+    const { keystone: userContext } = await getSchemaCtx('User')
+    const users = await User.getAll(userContext, { importRemoteSystem: SBBOL_IMPORT_NAME, deletedAt: null })
+
+    if (isEmpty(users)) return logger.info('No users imported from sbbol found. Cancel sync transactions')
+
+    for (const user of users) {
+        await requestTransactions(user.id, date)
+    }
+}
+
+module.exports = {
+    syncSbbolTransactionsCron: createCronTask('syncSbbolTransactionsCron', '0 0 * * *', async () => {
+        logger.info('Daily transaction synchronization started')
```

Comment:
It's redundant, because we have default logging in '@keystone/tasks' module.
I've created Saved Search in ELK
https://kibana-logs.in.doma.ai/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-3M,to:now))&_a=(columns:!(task.name,task.opts.repeat.cron,message),filters:!(('$state':(store:appState),exists:(field:task.name),meta:(alias:!n,disabled:!f,index:'96786050-22c0-11ed-8627-212f76435941',key:task.name,negate:!f,type:exists,value:exists))),index:'96786050-22c0-11ed-8627-212f76435941',interval:auto,query:(language:kuery,query:''),savedQuery:'All%20tasks',sort:!())